shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -69,6 +69,9 @@
   private static final Logger LOG = LoggerFactory.getLogger(CpCommand.class);
   private static final String COPY_SUCCEED_MESSAGE = "Copied %s to %s";
   private static final String COPY_FAIL_MESSAGE = "Failed to copy %s to %s";
+  private static final int OPTION_NOT_SET = -1;
```

Comment:
nit: i don't like this field. instead of mBufferSizeOption and mThreadOption as a field, why not have mBufferSize and mThread (set to default or value)?

Reply:
I refactored the fields.